src/icons/LogoHeartConfigurable.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from 'react'
+import Svg, {
+  ClipPath,
+  Defs,
+  G,
+  LinearGradient,
+  Path,
+  RadialGradient,
+  Stop,
+  SvgProps,
+} from 'react-native-svg'
+
+interface Props {
+  size?: number
+  testID?: string
+  color?: SvgProps['color']
```

Comment:
i would suggest restricting this prop to [Colors](https://github.com/valora-inc/wallet/blob/4e75656a53707685c19c5e8eed64d009a69cc752/src/styles/colors.tsx#L2) enum members to enforce only design system colors

Reply:
Fixed by replacing the component with original `LogoHeart` and adding `color` prop as per the discussion in the first comment of the PR.